### PR TITLE
Bugs fixed.

### PR DESCRIPTION
**Pull request by paveldanek**
- Needed to return the order number in addProduct, in order to display it.
- Some small enhancements to support readability of output.

### CODE REVIEW

**Review by rkeynguyen — APPROVED** [review]
Looks good

File: src/store/facade/GroceryStore.java [hunk 1]
```diff
@@ -481,14 +484,8 @@ public Result addProduct(Request request) {
...
 		String productId = productsList.add(new Product(request.getProductName(), request.getProductId(),
 				request.getProductCurrentPrice(), request.getProductStockOnHand(), request.getProductReorderLevel()));
-
 		if (!productId.equalsIgnoreCase("")) {
-
-			result.setResultCode(Result.ACTION_SUCCESSFUL);
-
-			result.setProductFields(productsList.searchById(productId));
-
-			reorderProduct(productsList.searchById(request.getProductId()));
```
Comment (carterclark) [inline src/store/facade/GroceryStore.java:491]: Why was this removed?
